README.md

```diff
@@ -111,6 +111,56 @@ public class PersonController {
 }
 ```
 
+* Simplify your controller keeping it fin
```

Comment:
What is meant by `fin` :)?

Reply:
It's just a typo 🤦 

I fixed it.

Thanks!